.github/workflows/backport.yml

```diff
@@ -0,0 +1,185 @@
+name: backport
+
+on:
+  pull_request_target:
+    types: [closed]
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
+
+jobs:
+  backport:
+    if: >
+      github.repository_owner == 'tarantool' &&
+      (
+        (
+          github.event_name == 'pull_request_target' &&
+          github.event.pull_request.merged
+        ) || (
+          github.event_name == 'issue_comment' &&
+          github.event.issue.pull_request &&
+          startsWith(github.event.comment.body, '/backport')
+        )
+      )
+
+    runs-on: ubuntu-20.04-self-hosted
+
+    timeout-minutes: 60
+
+    container:
+      image: docker.io/tarantool/testing:ubuntu-focal
+      # Mount /dev to the container to be able to mount a disk image inside it
+      # for successful run of the .github/actions/environment action.
+      volumes:
+        - /dev:/dev
```

Comment:
I vote for using a GitHub shared runner:

```yaml
runs-on: ubuntu-latest
timeout-minutes: 10
```

The reason is that we're using third party actions and it feels a bit unsafe to run uncontrolled code on our own runners.

Using GitHub shared runners seems OK here, because we don't do anything heavy in this job: no building, no testing. Just a checkout and a couple of API requests.

This way we also don't need `container:` here.

Also, we don't need the `tarantool/actions/prepare-checkout` step, because it is for self-hosted runners.

Is there anything against it?

Reply:
The alternative is to copy only needed code from these actions, carefully read it and place to https://github.com/tarantool/actions.

Pros:

* Allows to use it in the closed source projects.
* Allows to add features without depending on the upstream project.

Cons:

* Duplicates effort that is already done by GitHub users community.
* Complicates downstreaming a useful feature if we create it for ourself.

Reply:
> I vote for using a GitHub shared runner:

> runs-on: ubuntu-latest
> timeout-minutes: 10

But then we won't be able to use this action in tarantool-ee, no?

Reply:
> But then we won't be able to use this action in tarantool-ee, no?

That's true.

Reply:
While this topic is controversial, my initial comment is solved and so I'm going to close this discussion. Feel free to reopen if there are things to discuss further.